Source/WebCore/rendering/RenderObject.cpp

```diff
@@ -2645,6 +2645,29 @@ static void adjustTextDirectionForCoalescedGeometries(const SimpleRange& range,
     }
 }
 
+static bool currentNodeRequiresToSeparateBoundary(const RenderObject* currentRenderer)
```

Comment:
…and then this would become `currentNodeIsOutOfFlow` or `currentNodeIsOutOfFlowPositioned`